src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/LoginThrottlingFactory.php

```diff
@@ -49,7 +51,7 @@ public function addConfiguration(NodeDefinition $builder)
     {
         $builder
             ->children()
-                ->scalarNode('limiter')->info('The name of the limiter that you defined under "framework.rate_limiter".')->end()
+                ->scalarNode('limiter')->info(sprintf('A service id extending from "%s".', AbstractRequestRateLimiter::class))->end()
```

Comment:
should talk about the interface instead.

Reply:
Fixed

status: needs review